packages/env/src/web/index.ts

```diff
@@ -26,9 +26,16 @@ export const env = createEnv({
     AUTH_GOOGLE_SECRET: z.string().optional(),
     AUTH_GOOGLE_CODE_VERIFIER: z.string().optional(),
   },
-  client: {},
+  client: {
+    NEXT_PUBLIC_POSTHOG_KEY: z.string().optional(),
+    NEXT_PUBLIC_POSTHOG_HOST: z.string().optional(),
+  },
   experimental__runtimeEnv: {
     NODE_ENV: process.env.NODE_ENV,
+    // eslint-disable-next-line turbo/no-undeclared-env-vars
```

Comment:
You can disable this rule entirely for this package in the `eslint.config.js` file of `packages/env` because it won't ever work as that the turbo plugin wouldn't be able to see the `turbo.json` files of the apps, where these env vars are passed through.

Add this to `packages/env/eslint.config.js`:

```js
import { configs, defineConfig } from '@orbitkit/eslint';

export default defineConfig(...configs.base, {
  rules: {
    // We disable this rule in this package because turbo can't find the turbo.json files of the apps to know that we are actually passing through env vars there.
    'turbo/no-undeclared-env-vars': 'off',
  },
});
```

Then you can remove the `eslint-disable-next-line` comments from this file.